fix-code-2/main.cpp

```diff
@@ -3,6 +3,6 @@
 using namespace std;
 
 int main() {
-  string greeting = my name is Daniel and I am pleased to meet you!;
-  cout << s << endl;
+  string greeting = "my name is Daniel and I am pleased to meet you!";
+  cout << endl;
```

Comment:
this can be on line 6 . reduces lines of code

Reply:
Done.